doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -395,6 +395,15 @@ a.sk-footer-funding-link:hover {
   width: 100%;
 }
 
+/* Enables section links to be visible when linked */
+section[id]::before {
+  display: block;
+  height: 52px;
+  margin-top: -52px;
```

Comment:
I prefer keeping the CSS simple and use 52px, because most maintainers are not front-end experts.

BTW, if we want to use more "advanced CSS", I would go with [CSS variables](https://www.w3schools.com/css/css3_variables.asp), which all the major browsers supported in [2017](https://caniuse.com/css-variables).